src/db/spanner/schema.ddl

```diff
@@ -11,25 +11,28 @@
 --       ALSO, CONSOLE WANTS ONE SPACE BETWEEN DDL COMMANDS
 
 CREATE TABLE user_collections (
-  fxa_uid STRING(MAX)  NOT NULL,
-  fxa_kid STRING(MAX)  NOT NULL,
-  collection_id INT64  NOT NULL,
-  modified TIMESTAMP   NOT NULL,
+  fxa_uid STRING(MAX) NOT NULL,
+  fxa_kid STRING(MAX) NOT NULL,
+  collection_id INT64 NOT NULL,
+  modified TIMESTAMP  NOT NULL,
+
+  count INT64,
+  total_bytes INT64,
 ) PRIMARY KEY(fxa_uid, fxa_kid, collection_id);
 
 CREATE TABLE bsos (
-  fxa_uid STRING(MAX)  NOT NULL,
-  fxa_kid STRING(MAX)  NOT NULL,
-  collection_id INT64  NOT NULL,
-  bso_id STRING(MAX)   NOT NULL,
+  fxa_uid STRING(MAX) NOT NULL,
+  fxa_kid STRING(MAX) NOT NULL,
+  collection_id INT64 NOT NULL,
+  bso_id STRING(64)   NOT NULL,
```

Comment:
@jrconlin was this from the dev env? This should be `MAX` which is what stage's syncdb4 has.

```suggestion
  bso_id STRING(MAX)   NOT NULL,
```